libs/coin-modules/coin-polkadot/src/types/signer.ts

```diff
@@ -8,6 +8,7 @@ export type PolkadotSignature = {
   return_code: number;
 };
 export interface PolkadotSigner {
-  getAddress(path: string, verify?: boolean): Promise<PolkadotAddress>;
-  sign(path: string, message: string): Promise<PolkadotSignature>;
+  getAddress(path: string, ss58prefix: number, showAddrInDevice: boolean): Promise<PolkadotAddress>;
+  sign(path: string, message: Uint8Array, metadata: string): Promise<PolkadotSignature>;
+  getMajorVersion(): Promise<number>;
```

Comment:
It seems unnecessary to define this function as it is not used in `coin-polkadot` package.

Reply:
Fixed